dirty_cat/test/test_similarity_encoder.py

```diff
@@ -90,11 +90,25 @@ def test_similarity_encoder():
                 _test_similarity('ngram', string_distances.ngram_similarity,
                                  categories=category, n_prototypes=i)
                 _test_similarity('ngram', string_distances.ngram_similarity,
-                                 hashing_dim=2**16, categories=category,
+                                 hashing_dim=2 ** 16, categories=category,
                                  n_prototypes=i)
 
 
 def test_kmeans_protoypes():
     X_test = np.array(['cbbba', 'baaac', 'accc'])
-    proto = get_kmeans_protoypes(X_test, 3)
+    proto = get_kmeans_prototypes(X_test, 3)
     assert np.array_equal(np.sort(proto), np.sort(X_test))
+
+
+def test_reproducibility():
+    sim_enc = similarity_encoder.SimilarityEncoder(categories='k-means', n_prototypes=10)
+    X = np.array([' %s ' % chr(i) for i in range(32, 127)]).reshape((-1, 1))
+    prototypes = sim_enc.fit(X).categories_[0]
+    for i in range(10):
+        assert (not np.array_equal(prototypes, sim_enc.fit(X).categories_[0]))
```

Comment:
Don't do this part of the test: you might find centers that match by chance. Just leave the second part.